client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-panel.tsx

```diff
@@ -0,0 +1,40 @@
+import useSectionPatternsMapByCategory from './hooks/use-patterns-map-by-category';
+import PatternSelector from './pattern-selector';
+import type { Pattern, Category } from './types';
+import './pattern-list-panel.scss';
+
+type PatternListPanelProps = {
+	onSelect: ( selectedPattern: Pattern | null ) => void;
+	patterns: Pattern[];
+	selectedPattern: Pattern | null;
+	categories: Category[];
+	selectedCategory: string | null;
+	onDoneClick?: () => void;
+};
+
+const PatternListPanel = ( {
+	onSelect,
+	patterns,
+	selectedPattern,
+	categories,
+	selectedCategory,
+}: PatternListPanelProps ) => {
+	const sectionPatternsMapByCategory = useSectionPatternsMapByCategory( patterns, categories );
```

Comment:
Looks like we can simply pass `PatternsMapByCategory` to this component. WDYT?

Reply:
Nice 👍